src/Modules/ContentBlocks/templates/Frontend/Widgets/Detail.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if content_block is defined and content_block.text %}
+  <div class="module-content-blocks widget-content-blocks-default contentBlock{{ content_block.id }}">
```

Comment:
Wouldn't be more logical to store the id in a data-attribute? 
Or set the id-attribute?
And if a class is more logical, why not following the same pattern `content-block-$id`, instead of the snakeCase?

Reply:
I got this from the previous version so I don't know what the reasoning behind it was. Added an id attribute and changed the class to content-block-$id -> https://github.com/forkcms/forkcms/pull/3555/commits/6d98abe707acc811887e7dbf03fc45f7d7226998